doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/project-asset-execution-options-ref.adoc

```diff
@@ -0,0 +1,66 @@
+[id='project-asset-execution-options-ref_{context}']
+= Asset execution options with {PRODUCT}
+
+After you build and deploy your {PRODUCT} project to {KIE_SERVER} or other environment, you can execute the deployed assets for testing or for runtime consumption. You can also execute assets locally in addition to or instead of executing them after deployment.
```

Comment:
not sure if asset is the right word here... it looks that we're mixing kjar with other asset types like DRL, DMN, etc.

Reply:
Assets seems right to me. I consistently use "assets" to mean rules, processes, etc (DRL, DMN, BPMN), which are what we are executing. For kjar, I say "project" or "KJAR" specifically. You and I have spoken about this before, and I've gathered your feedback and that of others, and the consensus that we've run with in our docs is to refer to "projects" or "KJAR" files as we we deploy, and "assets" as what we author and execute individually to run what we've authored. I understand that there are nuances and technicalities, but from a user's perspective, we need to be clear and consistent, which is what we're trying to do.

Reply:
The problem is that you don't deploy assets, you deploy KJARs. So it sounds very confuse to me.

Reply:
Sure. I'll just say "execute the assets" then instead of "execute the deployed" assets.